.github/workflows/nightly_cron.yml

```diff
@@ -0,0 +1,79 @@
+name: Test against latest dependencies
+
+# Run if we change any of these paths and every night at 1 (or 2) AM Central time
+on:
+  push:
+    branches-ignore:
+      - 'master'
+    paths:
+    - 'requirements.txt'
+    - '.github/workflows/nightly_cron.yml'
+
+  schedule:
+    - cron: '0 7 * * *'  # Every day at 07:00 UTC (1AM CST or 2AM CDT)
+
+jobs:
+  build:
+    name: Build against latest dependencies
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, macos-latest, ubuntu-latest]
+        python-version: ['3.8']
+        architecture: ['x86', 'x64']
+        exclude:
+          # Don't build 32-bit on Mac or Linux
+          - os: macos-latest
+            architecture: x86
+
+          - os: ubuntu-latest
+            architecture: x86
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Setting up Python
+        uses: actions/setup-python@v1.1.1
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: ${{ matrix.architecture }}
+
+      - name: Updating pip
+        run: python -m pip install --upgrade pip
+        shell: bash
+
+      - name: Collecting latest dependencies
+        id: dependencies  # Needed to retrieve the output of this step later
+        run: |
+          dependencies=$(python build_tools/github/get_latest_dependencies.py)
+          echo "::set-output name=latest_dependencies::${dependencies}"
+        shell: bash
+
+      # We install one-by-one because some dependencies depend on others
+      - name: Installing dependencies
+        run: |
+          for dependency in $DEPENDENCIES; do
+            python -m pip install $dependency
+          done
+        shell: bash
+        env:
+          DEPENDENCIES: ${{ steps.dependencies.outputs.latest_dependencies }}
+
+      - name: Building wheel
+        run: make bdist_wheel
+        shell: bash
+
+      - name: Installing generated wheel
+        run: pip install --pre --no-index --find-links dist/ pmdarima
+        shell: bash
+
+      - name: Running unit tests
+        run: |
+          if [ "${{ matrix.os }}" == "macos-latest" ]; then
+            export PMD_MPL_BACKEND=TkAgg
+          fi
+          pytest --showlocals --durations=20 --pyargs pmdarima --benchmark-skip
```

Comment:
FWIW, you prob don't need the `--benchmark-skip` since we no longer use pytest-benchmark.

Reply:
Cool, I will remove that!